src/components/SendButton/sendButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styles from './sendButton.module.css';
+import { IconContext } from 'react-icons';
+import { FiArrowRight } from 'react-icons/fi';
+
+interface Props {
+  status: string;
+}
+
+export const SendButton = ({ status }: Props) => {
```

Comment:
I appreciate this is partly just from picked up conventions on the other components (which I could have picked up before). I have a preference for doing this:
```
export const SendButton = (props: Props) => {
  const { status } = props;